src/itest/java/uk/gov/companieshouse/pscstatementdataapi/steps/PscStatementsSteps.java

```diff
@@ -125,8 +138,26 @@ public void i_send_psc_statement_get_request_with_statement_id(String companyNum
         CucumberContext.CONTEXT.set("getResponseBody", response.getBody());
     }
 
+    @When("I send a GET request for company number {string} to company metrics api")
+    public void i_send_get_request_to_company_metrics_api(String companyNumber) throws IOException {
+        String uri = "/company/{company_number}/metrics";
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("ERIC-Identity", "TEST-IDENTITY");
+        headers.set("ERIC-Identity-Type", "key");
+        headers.set("ERIC-Authorised-Key-Roles", "*");
+        HttpEntity<String> request = new HttpEntity<String>(null, headers);
+
+        ResponseEntity<MetricsApi> response = restTemplate.exchange(uri, HttpMethod.GET, request,
+                MetricsApi.class, companyNumber);
+
+        CucumberContext.CONTEXT.set("statusCode", response.getStatusCodeValue());
+        CucumberContext.CONTEXT.set("getResponseBody", response.getBody());
+    }
+
     @When("I send a GET statement list request for company number {string}")
     public void get_statement_list_for_company_number(String companyNumber) throws IOException {
+        File metricsFile = new ClassPathResource("/json/input/company_metrics.json").getFile();
```

Comment:
Is it worth using the company number in the name of the metrics file? Allows us to expand our calls if needed